portal/server/routes/contract/saveFacilityAndGoBack.js

```diff
@@ -0,0 +1,41 @@
+const {
+  requestParams,
+  errorHref,
+  postToApi,
+} = require('../../helpers');
+const api = require('../../api');
+const { formDataMatchesOriginalData } = require('./formDataMatchesOriginalData');
+
+const saveFacilityAndGoBackToDeal = async (req, res, sanitizedBody) => {
+  const { _id: dealId, bondId, loanId, userToken } = requestParams(req);
+  const facility = req.apiData.bond?.bond ?? req.apiData.loan?.loan;
```

Comment:
Should it be `onst facility = req.apiData?.bond.bond ?? req.apiData?.loan.loan;`?

Reply:
Good shout, apiData and bond/loan can both be undefined, will do `const facility = req.apiData?.bond?.bond ?? req.apiData?.loan?.loan;`